packages/sdk-js/src/GrowthBook.ts

```diff
@@ -551,7 +620,9 @@ export class GrowthBook<
           }
```

Comment:
for this:
```
if (result.value.urlRedirect && experiment.urlPatterns) {
```
consider using `experiment.changeType` instead. May have some stale payload delays before working though...

Reply:
no, we're stuck

Reply:
use helper